### DISCUSSION TITLE
Can I safely use this tool with vanguard anticheat?

### DISCUSSION THREAD

**Discussion by outcome23** [KevinBytesTheDust/CVAmp#279] [Q&A] [answered]
i got a question that might be a bit weird and out of topic.. im a gamer, and i was wondering, if the anticheating system vanguard that riot game uses, somehow can detect the program and might find the player of cheating which results a permaban.. im new into all this, and i just wanna be sure.. ty for the answers before hand

**Comment by kh2000bd** [comment, marked as answer]
You are not at risk at all of using this program with vanguard. The CTVB is essentially a stripped down version of Google chrome. "Essentially". But you should be fine.
